### DISCUSSION TITLE
How to display a countdown timer inside a local notification?

### DISCUSSION THREAD

**Discussion by akshat-triv** [ionic-team/capacitor#5952] [Q&A]
I've seen other android apps displaying a live countdown timer inside the notification body, and title as well. My application has a timer functionality user can start a countdown timer and I am using local notifications to notify the user when the timer has finished. I would like to show my user a notification with a live countdown timer init, displaying how much time is left. Currently, I don't see any way of updating the body of a delivered local notification.  Is there any way to achieve this functionality using Capacitor js Local Notifications or is there any 3rd party library I can use to do this?

**Comment by angeloraso** [comment]
Hi. Yes. It's possible. I don't think there is something specific for your problem. You will need to create a capacitor plugin to generate a native notification to display the countdown and take action when it reaches its end.

**Reply by akshat-triv** [reply]
Hi, Thanks for replying. Can anyone from the capacitor js community write this plugin, I'm not very experienced with native development. And I think this feature should be a part of Local and Push Notifications from capacitor js. A progress bar would also be a good addition inside a notification, to display how much a task has been completed.
